utensor_cgen/snippets.py

```diff
@@ -73,51 +73,56 @@ def _permute_args(args: list, perm: list=None):
 
 
 class AddOpSnippet(Snippet):
-  def __init__(self, inputs, output, tf_dtype=tf.float32):
+  def __init__(self, inputs, output, init_count, tf_dtype=tf.float32):
```

Comment:
I think we should set `init_count` a default value, say, 0.
And let the optimizer to decide the desired `init_count`.